webapp/src/component/SmoothProgress.tsx

```diff
@@ -1,15 +1,22 @@
-import { useState, useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { useDebounce } from 'use-debounce';
 import { styled } from '@mui/material';
 import clsx from 'clsx';
 
 const StyledProgress = styled('div')<{ loading?: string; finish?: string }>`
   height: 4px;
-  background: ${({ theme }) => theme.palette.primary.light};
+  background: ${({ theme }) => {
+    if (theme.palette.mode == 'dark') {
+      return theme.palette.primary.light;
+    }
+    return '#c9a2b5';
```

Comment:
Could you add this color to theme. Or add e.g. global loader color, which would be defined for both themes?